### PR TITLE
Fixed broken absolute links

### PR DESCRIPTION
**Pull request by n-boshnakov**
**What this PR does / why we need it**:
This PR fixes broken absolute links and removes broken links from the manifest
**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Release note**:

```improvement developer
NONE
```

### CODE REVIEW

File: website/documentation/tutorials/shoot_istio_dns_certs/_index.md [hunk 1]
```diff
@@ -222,7 +222,7 @@ I want to install Istio with a default profile and SDS enabled. Furthermore I pa
     dns.gardener.cloud/ttl: "120"
 ```
 With these annotations three things now happen automagically:
-1. The [External DNS Manager](https://gardener.cloud/docs/guides/install_gardener/setup/), provided to you as a service (`dns.gardener.cloud/class: garden`), picks up the request and creates the wildcard DNS entry `*.gsicdc.mydomain.io` with a time to live of 120sec at your DNS provider. My provider Cloud Flare is very very quick (as opposed to some other services). You should be able to verify the entry with `dig lovemygardener.gsicdc.mydomain.io` within seconds.
+1. The [External DNS Manager](https://gardener.cloud/docs/concepts/networking/dns-managment/), provided to you as a service (`dns.gardener.cloud/class: garden`), picks up the request and creates the wildcard DNS entry `*.gsicdc.mydomain.io` with a time to live of 120sec at your DNS provider. My provider Cloud Flare is very very quick (as opposed to some other services). You should be able to verify the entry with `dig lovemygardener.gsicdc.mydomain.io` within seconds.
```
Comment (Kristian-ZH) [inline website/documentation/tutorials/shoot_istio_dns_certs/_index.md:225]: I prefer on github page to be used a github links -  https://github.com/gardener/external-dns-management/blob/master/README.md

**Review by Kristian-ZH — CHANGES_REQUESTED** [review]
Can you please address these comments?

File: website/blog/2020-10/00/Gardener-Integrates-with-KubeVirt.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ Integration with KubeVirt follows the Gardener extensibility concept and introdu
...
+The **Infrastructure Controller** configures the network communication between the shoot worker nodes. By default, shoot worker nodes only use the provider cluster’s pod network. To achieve higher level of network isolation and better performance, it is possible to add more networks and replace the default pod network with a different network using container network interface (CNI) plugins available in the provider cluster. This is currently based on [Multus CNI](https://github.com/intel/multus-cni/blob/master/README.md) and [NetworkAttachmentDefinitions](https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/quickstart.md).
```
[the remark this answers is not in the document]
Comment (n-boshnakov) [reply website/blog/2020-10/00/Gardener-Integrates-with-KubeVirt.md:50]: Did you mean another link? You have given the exact same link I have placed the file.

[the remark this answers is not in the document]
Comment (Kristian-ZH) [reply website/blog/2020-10/00/Gardener-Integrates-with-KubeVirt.md:50]: my mistake, the link is correct

**Review by Kristian-ZH — DISMISSED** [review]
/lgtm

**Review by Kristian-ZH — APPROVED** [review]

**Pull request merged**